### PR TITLE
Enable Ci Debugger in bazel action

### PR DESCRIPTION
**Pull request by pat-trunk-io**

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -182,3 +190,26 @@ runs:
...
         BAZEL_NEGATIVE_TAG_FILTER: ${{ inputs.test-negative-tag-filter }}
         CI: "true"
+
+    - name: Test Impacted Targets (with debugger)
+      if: inputs.test-targets == 'true' && inputs.enable-trunk-ci-debugger
+      id: test-impacted-targets-dbg
+      uses: trunk-io/breakpoint@v1
+      with:
+        breakpoint-id: ${{ inputs.trunk-ci-debugger-breakpoint-name }}
+        trunk-token: ${{ env.TRUNK_TOKEN }}
```
Comment (TylerJang27) [inline action.yaml:200]: Don't we usually specify to use this from `secrets.`?

**Review by TylerJang27 — APPROVED** [review]
See one note. Also we can close this out https://linear.app/trunk/issue/TRUNK-10103/add-ci-debugger-support-to-bazel-action

File: action.yaml (hunk 1 above)
Comment (pat-trunk-io) [reply action.yaml:200]: Yes, but an action can't access secrets from the og repo, maybe I'll just add it as in input

Comment (pat-trunk-io) [reply action.yaml]: Actually completely missed that the action already takes a token as an input, modified

**Pull request merged**